Signal/src/ViewControllers/HomeViewController.m

```diff
@@ -490,13 +488,10 @@ - (void)viewDidAppear:(BOOL)animated
 {
     [super viewDidAppear:animated];
 
-    if (self.newlyRegisteredUser) {
-        [self markAllUpgradeExperiencesAsSeen];
```

Comment:
This was redundant with what happens in the appDelegate regsitrationStateDidChange notification observer.